airtime_mvc/application/controllers/PreferenceController.php

```diff
@@ -262,6 +262,41 @@ public function streamSettingAction()
                     //Application_Model_RabbitMq::PushSchedule();
                 }
 
+                // pulling this from the 2.5.x branch
+                if (!Application_Model_Preference::GetMasterDjConnectionUrlOverride()) {
+                    $master_connection_url = "http://".$_SERVER['SERVER_NAME'].":".$values["master_source_port"].$values["master_source_mount"];
+                    if (empty($values["master_source_port"]) || empty($values["master_source_port"])) {
```

Comment:
Yeah honestly this was just there from my attempt to integrate the override functionality before I abandoned it for this commit. The question is do we want to integrate the ability to override the URL ? It seemed like the functionality was primarily a edge-case with a firewall requiring the rewriting of the URL.